src/components/timeline/status.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Alert from './alert';
+import Spinner from './spinner';
+import ErrorMessageContainer from './error-message.container';
+
+const Status = ( { status } ) => {
+	switch ( status ) {
+		case 'fetching':
+			return <Spinner />;
+		case 'notready':
+			// @TODO Translate
+			return (
+				<Alert type="info">
+					Please provide a user and wiki to begin.
```

Comment:
Ticket mentions this as default. and when only one user is provided
 **Please provide two usernames and wiki to begin.**

NOTE: I'm not sure if we should display revisions of only one user if we use this message.

Reply:
So right now we actually will run the request with just one username and a wiki. While that isn't really all that useful, it's _technically_ all that is required (since now we are requesting each user individually).

I'm not really opposed to changing it waits until there are two users.

Maybe it would be best to let the fetch happen, but not display any results if there are not at least two users?